DictDataClasses.py

```diff
@@ -0,0 +1,75 @@
+import random
+import string
+import datetime
+
+class DictDataClasses:
+
+    def __init__(self):
+        self.__n_inst_ref = 14000
```

Comment:
What is this value for? Can you make it customisable through parameter passing? And then maybe have it default to 14000 otherwise?

Reply:
Was the numbers of identifers Raj said in #35